app/Console/Commands/AdsFetchHosts.php

```diff
@@ -145,7 +161,30 @@ private function handleBroadcast(Broadcast $broadcast): void
         $address = $broadcast->getAddress();
         $time = $broadcast->getTime();
 
-        NetworkHost::registerHost($address, $message, $time);
+        $info = null;
+        $infoUrl = $this->prepareInfoUrl($message);
+
+        try {
+            $info = $this->client->fetchInfo($infoUrl);
+        } catch (UnexpectedClientResponseException $exception) {
+            $this->info(sprintf('Demand server `%s` does not support `/info` endpoint.', $infoUrl));
+        } catch (RuntimeException $exception) {
+            $this->error(sprintf('Could not import info data (%s) from server `%s`.', $exception->getMessage(), $infoUrl));
+        }
+
+        NetworkHost::registerHost($address, $message, $info, $time);
+    }
+
+    private function prepareInfoUrl(string $hostUrl): string
+    {
+        $parsedUrl = parse_url($hostUrl);
+        $path = $parsedUrl['path'] ?? '';
+
+        if ($path === '/info') {
+            return $path;
```

Comment:
I suppose that this is a bug.
```suggestion
            return $hostUrl;
```

Reply:
yes :)